api/v1alpha1/nodenetworkconfig_types.go

```diff
@@ -32,7 +32,8 @@ type NodeNetworkConfigSpec struct {
 
 // NodeNetworkConfigStatus defines the observed state of NodeConfig.
 type NodeNetworkConfigStatus struct {
-	ConfigStatus string `json:"configStatus"`
+	ConfigStatus     string      `json:"configStatus"`
+	LastModification metav1.Time `json:"lastModification"`
```

Comment:
Can we call this `lastApplied`? Or what exactly is it supposed to express? Some Comments on the types would be good as they also appear in the CRD.

Reply:
Changed that to `lastUpdated`.
This stores the date/time when NodeNetworkConfigStatus was last updated. Is later used with `configTimeout` to invalidate config if node is unresponsive for some reason and status is not changed to `provisioned` before timeout.